Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/how-to-deploy-a-zkapp.mdx

---
title: How to Deploy a zkApp
hide_title: true

> description: A step-by-step guide on how to deploy a zkApp to the Berkeley Testnet. The guide covers configuring your smart contract project, requesting funds from the faucet, and deploying your smart contract.

Reviewer: ```suggestion
description: Guided steps to deploy a zkApp to the Berkeley Testnet to configure your smart contract project, request funds from the faucet, and deploy your smart contract.
```

---
title: How to Deploy a zkApp
hide_title: true
description: A step-by-step guide on how to deploy a zkApp to the Berkeley Testnet. The guide covers configuring your smart contract project, requesting funds from the faucet, and deploying your smart contract.
keywords:
  - zkApp

>   - Deployment

Reviewer: ```suggestion
  - deploy a smart contact
```

Reviewer: wdyt about using phrases the way users will search?

Author: I like it!

>   - Berkeley Testnet

Reviewer: ```suggestion
```

Reviewer: oh hmmm, do we want to put a testnet name into the keywords? In this case, the topic covers the steps to deploy to Berkley Testnet... we'll need to remove all of the Berkley Testnet mentions after hard fork, right? (what comes after testnet? mainnet?)

Author: Good catch! Lets not deal with the headache of removing it later and just do it now! :D

>   - Smart contract
>   - Configuration

Reviewer: ```suggestion
  - smart contract
  - configure a zkapp
```

Reviewer: the few models that I've look at use lower case (probably keywords are case-insensitive?)
